### PR TITLE
Backport trivial pr19

### PR DESCRIPTION
**Pull request by PastaPastaPasta**

### CODE REVIEW

File: src/qt/walletview.cpp [hunk 1]
```diff
@@ -20,6 +20,7 @@
 #include <qt/walletmodel.h>
 
 #include <interfaces/node.h>
+#include <psbt.h>
```
Comment (PastaPastaPasta) [inline src/qt/walletview.cpp:23]: Might need to drop 11... Weird that it didn't do the other part of the commit, yet merged smoothly

File: src/test/sanity_tests.cpp [hunk 2]
```diff
@@ -14,7 +14,7 @@ BOOST_AUTO_TEST_CASE(basic_sanity)
 {
   BOOST_CHECK_MESSAGE(glibc_sanity_test() == true, "libc sanity test");
   BOOST_CHECK_MESSAGE(glibcxx_sanity_test() == true, "stdlib sanity test");
-  BOOST_CHECK_MESSAGE(ECC_InitSanityCheck() == true, "openssl ECC test");
+  BOOST_CHECK_MESSAGE(ECC_InitSanityCheck() == true, "secp256k1 sanity test");
```
Comment (UdjinM6) [inline src/test/sanity_tests.cpp:17]: 18825 :
>OpenSSL is long gone.

Is it?

File: configure.ac [hunk 3]
```diff
@@ -385,6 +385,7 @@ if test "x$enable_werror" = "xyes"; then
   if test "x$CXXFLAG_WERROR" = "x"; then
     AC_MSG_ERROR("enable-werror set but -Werror is not usable")
   fi
+  AX_CHECK_COMPILE_FLAG([-Werror=gnu],[ERROR_CXXFLAGS="$ERROR_CXXFLAGS -Werror=gnu"],,[[$CXXFLAG_WERROR]])
```
Comment (UdjinM6) [inline configure.ac:388]: 18887: probably requires 18088 (and fixes mentioned there)

File: doc/productivity.md [hunk 4]
```diff
@@ -12,7 +12,7 @@ Table of Contents
    * [Multiple working directories with `git worktrees`](#multiple-working-directories-with-git-worktrees)
    * [Interactive "dummy rebases" for fixups and execs with `git merge-base`](#interactive-dummy-rebases-for-fixups-and-execs-with-git-merge-base)
 * [Writing code](#writing-code)
-   * [Format C/C++/Protobuf diffs with `clang-format-diff.py`](#format-ccprotobuf-diffs-with-clang-format-diffpy)
+   * [Format C/C++ diffs with `clang-format-diff.py`](#format-cc-diffs-with-clang-format-diffpy)
```
Comment (UdjinM6) [inline doc/productivity.md:15]: 19058: requires 17165

File: doc/build-windows.md [hunk 5]
```diff
@@ -9,7 +9,7 @@ The options known to work for building Dash Core on Windows are:
 and is the platform used to build the Dash Core Windows release binaries.
 * On Windows, using [Windows
 Subsystem for Linux (WSL)](https://docs.microsoft.com/windows/wsl/about) and the Mingw-w64 cross compiler tool chain.
-* On Windows, using a native compiler tool chain such as [Visual Studio](https://www.visualstudio.com).
+* On Windows, using a native compiler tool chain such as [Visual Studio](https://www.visualstudio.com). See [README.md](/build_msvc/README.md).
```
Comment (UdjinM6) [inline doc/build-windows.md:12]: 19061: there is no such file

File: src/qt/walletview.cpp (hunk 1 above)
Comment (UdjinM6) [reply src/qt/walletview.cpp:23]: agree, let's drop it for now

File: doc/REST-interface.md [hunk 6]
```diff
@@ -72,7 +72,6 @@ Only supports JSON as output format.
 * pruned : (boolean) if the blocks are subject to pruning
 * pruneheight : (numeric) highest block available
 * softforks : (array) status of softforks in progress
-* bip9_softforks : (object) status of BIP9 softforks in progress
```
Comment (UdjinM6) [inline doc/REST-interface.md:75]: 19390: too early for this

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

**Comment by PastaPastaPasta** [comment]
dropped all and squash 17994 fix

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**